contracts/governance/StakingRewards/StakingRewards.sol

```diff
@@ -82,7 +83,7 @@ contract StakingRewards is StakingRewardsStorage, Initializable {
 			require(currentTS > withdrawals[staker], "allowed after 14 days");
 		}
 
-		for (uint256 i = withdrawals[staker]; i < currentTS && i < withdrawals[staker].add(maxDuration); i += TWO_WEEKS) {
+		for (uint256 i = withdrawals[staker]; i <= currentTS && i < withdrawals[staker].add(maxDuration); i += TWO_WEEKS) {
```

Comment:
the period is considered passed the next second after the 2-weeks multiple reached, not on the same second
```suggestion
		for (uint256 i = withdrawals[staker]; i < currentTS && i < withdrawals[staker].add(maxDuration); i += TWO_WEEKS) {
```